src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method completeOrder was called. Params: userId={}", userId);
```

Comment:
I think here we must used .info. Because info used when your program is working as expected, but debug used when we find the reason in case your program is not working as expected.